source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,539 @@
 .. _vmware-esxi:
 
-Use VMware\* ESXi guest
-#######################
+Run Clear Linux as a VMware\* ESXi guest OS
+###########################################
 
+`VMware ESXi`_ is a type 1 bare-metal hypervisor which runs directly on top of 
+server hardware.  With VMware ESXi, you can create, configure, manage, and run 
+|CLOSIA| virtual machines in the cloud.  
 
-This section is based on *VMware vSphere 6* and explains how to use |CLOSIA|
-as ESXi Guest.
+This section explains 2 methods on how to create and run |CL| in a virtualized 
+environment using VMware ESXi 6.5 Update 1:  
 
-Ensure `Intel® Virtualization Technology`_ (Intel® VT) and
-`Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) are both
-enabled in your BIOS/UEFI firmware configuration.
+* :ref:`label-method-1-vmware-esxi`.  This method provides flexibility 
+  in configuring, for example: its size, the nummber of partitions, the initial 
+  selection of bundles, etc.
+* :ref:`label-method-2-vmware-esxi`.  This method uses a pre-configured VMware 
+  |CL| image.
 
-Run Clear Linux OS for Intel Architecture
-=========================================
+.. note::
 
-#. Download the `latest`_ |CL| **live** version
-   :file:`clear-{release-number}-live.img.xz`.
+  VMware also offers a type 2 hypervisor called `VMware Workstation Player`_ which 
+  is designed for the Desktop environment.  For information on how on run |CL| 
+  on it, see :ref:`vmware-player`.
 
-#. Decompress the downloaded image. Uncompressed image size is ~ **5GB**.
+.. _label-method-1-vmware-esxi:
 
-   * On Linux:
+Method 1: Install Clear Linux into a new :abbr:`VM (Virtual Machine)` 
```

Comment:
Do not use abbreviations on headings. Consider putting each method on their own file.